trade-finance-manager-ui/templates/case/underwriting/underwriting.njk

```diff
@@ -9,6 +9,8 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} - Underwriting{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.